src/Microsoft.DotNet.Darc/src/DarcLib.AzDev/AzureDevOpsClient.cs

```diff
@@ -446,6 +447,75 @@ public async Task MergePullRequestAsync(string pullRequestUrl, MergePullRequestP
                 repoName,
                 id);
         }
+        public async Task<IList<Commit>> GetPullRequestCommitsAsync(string pullRequestUrl)
+        {
+            (string accountName, string projectName, string repoName, int id) = ParsePullRequestUri(pullRequestUrl);
+            VssConnection connection = CreateVssConnection(accountName);
```

Comment:
I'm betting both the VssConnection and the GitHttpClient are disposable, and should be disposed.

Reply:
I was looking at this, like no where in the file we dispose connection object, is that for a reason?

Reply:
Because a lot of this code got created before I was code reviewing this repository.  Every usage of "VssConnection" and "GitHubClient" should be disposed.

Reply:
okay will dispose the connection object..

Reply:
Could you create an issue to review these and fix them? it's not a good idea to leak these.

Reply:
https://github.com/dotnet/core-eng/issues/10248